### PR TITLE
Add support for traditional mode test

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 virsh.remote_with_unix --vt-connect-uri qemu:///system
JOB ID     : f199d187de33f5ababbf1172831a72935efe29ad
JOB LOG    : /root/avocado/job-results/job-2020-06-28T06.19-f199d18/job.log
 (01/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.default_unix_config: PASS (51.30 s)
 (02/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.readonly: PASS (49.95 s)
 (03/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.customized_unix_sock_dir: PASS (111.74 s)
 (04/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.default_config_and_start_vm: PASS (52.27 s)
 (05/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.allow_sasl_krb_user: PASS (157.44 s)
 (06/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.allow_sasl_users: PASS (134.00 s)
 (07/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.socket_access_controls: PASS (56.15 s)
 (08/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.socket_with_auth_conf: PASS (119.60 s)
 (09/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.socket_with_polkit_and_acl_control: PASS (53.12 s)
 (10/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.positive_testing.socket_with_polkit_rw: PASS (51.63 s)
 (11/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.no_allowed_sasl_user: PASS (170.99 s)
 (12/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.allow_sasl_user_with_readonly: PASS (169.02 s)
 (13/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.readonly_without_auth: PASS (51.31 s)
 (14/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.socket_access_controls.cfg_file: PASS (54.85 s)
 (15/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.socket_access_controls.socket_file: PASS (50.99 s)
 (16/16) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.socket_with_polkit_ro: PASS (52.13 s)
RESULTS    : PASS 16 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1388.87 s

File: virttest/utils_conn.py [hunk 1]
```diff
@@ -1697,8 +1702,16 @@ def conn_recover(self):
         try:
             libvirtd_service = utils_libvirtd.Libvirtd(session=client_session)
             if self.libvirt_ver:
-                process.run("systemctl daemon-reload")
-                process.run("systemctl stop libvirtd.socket")
+                # For traditional mode, need unmask libvirtd*.socket
+                if traditional_mode == 'yes':
+                    process.run("systemctl unmask libvirtd.socket")
```
Comment (chunfuwen) [inline virttest/utils_conn.py:1707]: Better add shell=True when call process.run()

File: virttest/utils_conn.py [hunk 2]
```diff
@@ -1688,6 +1691,8 @@ def conn_recover(self):
         (1).Delete remote file.
         (2).Restart libvirtd on server.
         """
+        traditional_mode = self.traditional_mode
```
Comment (chunfuwen) [inline virttest/utils_conn.py:1694]: what it means 'traditional_mode' ? is it means mode for starting libvirtd ? monolith libvirtd vs modular libvirtd?
As far as I know, currently we still use  monolith libvirtd  to start libvirtd, so it should be default one.

**Comment by chunfuwen** [comment]
@yafu-1 , please help confirm this

File: virttest/utils_conn.py (hunk 1 above)
Comment (cliping) [reply virttest/utils_conn.py]: Updated. Thanks.

File: virttest/utils_conn.py (hunk 2 above)
Comment (cliping) [reply virttest/utils_conn.py:1694]: Yes, it is means monolith libvirtd.

Comment (pevogam) [reply virttest/utils_conn.py:1694]: The meaning of this term or rather its scope would best be explained within the code for people that will not understand it. Just a remark here.

**Pull request merged**